docs/guides/in-product-word-list.mdx

```diff
@@ -0,0 +1,522 @@
+
+Use **copy** when Jobber creates a new version that would have no effect on the
+original when edited.
+
+| **✅ Do**  | **❌ Don't**    |
+| ---------- | --------------- |
+| Copy Quote | Duplicate Quote |
+
+Use **attach** when describing receipts, forms or signatures being added to an
+object. For file selection, use **select a file** for consistency.
+
+| **✅ Do**                                                  | **❌ Don't**                                               |
+| ---------------------------------------------------------- | ---------------------------------------------------------- |
+| Attach a receipt to this expense <br /> [Select a file](#) | Attach a receipt to this expense <br /> [Attach a file](#) |
+
+<br />
+
+### save / update / submit
+
+Use **save** at the end of a form on completion or **save changes** on editing.
+
+| **✅ Do**               | **❌ Don't** |
+| ----------------------- | ------------ |
+| Save Job / Save Changes | Update Job   |
+
+Use **update** when individual changes or groups of changes can be saved inline
+without needing to save a whole view or page.
+
+| **✅ Do**            | **❌ Don't**       |
+| -------------------- | ------------------ |
+| Update Company Hours | Save Company Hours |
+
+Use **submit** when an account user is completing and sending a form to Jobber,
+or when a service consumer is submitting a form to their service provider.
+
+| **✅ Do**       | **❌ Don't**  |
+| --------------- | ------------- |
+| Submit Feedback | Save Feedback |
+
+<br />
+
+### sync / pull / push
+
+Use sync when a process with integration is active or to describe an integration
+in a general way.
+
+| **✅ Do**                            | **❌ Don't**                         |
+| ------------------------------------ | ------------------------------------ |
+| It appears the sync may have stalled | It appears the push may have stalled |
+
+Use **pull** or **push** when the sync is explicitly one-way with any aspect of
+an integration.
+
+| **✅ Do**                                              | **❌ Don't**                                           |
+| ------------------------------------------------------ | ------------------------------------------------------ |
+| Push approved timesheet hours                          | Sync approved timesheet hours                          |
+| All active items will be pulled from QuickBooks Online | All active items will be synced from QuickBooks Online |
+
+<br />
+
+### vault / vaulted / add / store
+
+Avoid using **vault** or **vaulted** for card storage. Use **add** and **store**
+instead.
+
+| **✅ Do**                              | **❌ Don't**                                      |
+| -------------------------------------- | ------------------------------------------------- |
+| Stored cards will need to be re\-added | Any vaulted cards will need to be re\-established |
+
+<br />
+
+## Concepts and Items
```

Comment:
Maybe a general rule under this one about capitalization of Jobber concepts/items? I've seen Jobs, invoices, etc get capitalized sometimes (outside of page titles and menu item titles and things of that nature which warrant it)

Reply:
Although this is a vocabulary document and not necessarily a grammatical doc

Reply:
(This is not a specifically requested change as you can tell by my back-and-forth comments)

Reply:
I will add this note here for now anyway until we pull stuff elsewhere as needed